plugins/modules/instance.py

```diff
@@ -234,6 +234,12 @@
     ),
 )
 
+spec_additional_ipv4 = dict(
+    public=SpecField(
+        type=FieldType.bool, description="fill this in", required=True
```

Comment:
Looks like the description still needs to be written here

Reply:
Good catch, totally forgot about this!